### DISCUSSION TITLE
无法访问开往页面

### DISCUSSION THREAD

**Discussion by Pinpe** [travellings-link/travellings#1623] [寻求帮助] [answered]
www.travellings.cn无法访问了。
![屏幕截图 2023-07-22 003018.png](https://img1.imgtp.com/2023/07/22/AaV7D1jz.png)

**Comment by linlinzzo** [comment]
您好，目前开往域名 travellings.cn 正在进行 ICP 备案主体的变更，目前处于关站状态。您可以通过访问 travellings.xiaozonglin.cn 来临时开往新的网站。我们会在网站恢复后第一时间通知您。抱歉给您造成不便。

详见 https://blog.travellings.cn/posts/icp-closed

**Comment by linlinzzo** [comment, marked as answer]
开往的部分服务已恢复，您可以尝试访问 https://www.travellings.cn/ 。如果您发现开往的服务问题或者想对开往提出意见，欢迎您再次使用 Discussion。
